openml-utils/src/test/java/com/feedzai/openml/util/data/encoding/EncodingHelperTest.java

```diff
@@ -222,4 +233,28 @@ private void validateNumericFieldEncoding(final EncodingHelper encodingHelper, f
                 .as("Encoding a string")
                 .isEqualTo(Double.NaN);
     }
+
+    /**
+     * Tests the existing encoders from {@link EncodingHelper#encoderForField(AbstractValueSchema)}.
+     */
+    @Test
+    public void testEncoders() {
```

Comment:
"testEncoders" but then tests only for null values, seems like a bit of a mis-match in terms of description -> content, no ?

Reply:
true. will rename to `testEncodersOnNulls` or something like that